### DISCUSSION TITLE
Activations of every layer

### DISCUSSION THREAD

**Discussion by neural-everything** [lululxvi/deepxde#1515] [Q&A]
Hi @lululxvi ,
Is there a way to get the activation values at every layer of the network? I am trying to investigate the effect of scaling the PDE on training behaviour. Thanks.

**Comment by lululxvi** [comment]
Use `Model.print_model()` or `Model.state_dict()`

**Reply by ruchitkini** [reply]
Hi, 
I tried using ``` model.print_model()``` command but I am getting the following error: ``` NotImplementedError: state_dict hasn't been implemented for this backend. ``` I am using pytorch as the backend. I am using this command after I call the stored trained model.

**Reply by lululxvi** [reply]
Use `Model.state_dict()`
